src/mame/drivers/drgnmst.cpp

```diff
@@ -46,27 +46,23 @@ DSW2    68000      TPC1020           62256
 #include "speaker.h"
 
 
-WRITE16_MEMBER(drgnmst_state::drgnmst_coin_w)
+WRITE16_MEMBER(drgnmst_state::coin_w)
 {
 	machine().bookkeeping().coin_counter_w(0, data & 0x100);
 	machine().bookkeeping().coin_lockout_w(0, ~data & 0x400);
 	machine().bookkeeping().coin_lockout_w(1, ~data & 0x800);
 }
 
-WRITE16_MEMBER(drgnmst_state::drgnmst_snd_command_w)
+WRITE8_MEMBER(drgnmst_state::snd_command_w)
 {
-	if (ACCESSING_BITS_0_7)
-	{
-		m_snd_command = (data & 0xff);
-		m_maincpu->yield();
-	}
+	m_snd_command = (data & 0xff);
```

Comment:
Now that this is `WIRTE8` the `& 0xff` is redundant (it's already definitely no larger than that).